### ISSUE TITLE
Add HTTP/S / SOCKS proxy support to avoid invalid cookies

### ISSUE THREAD

**Issue by JReverse** [FlareSolverr/FlareSolverrSharp#4]
Add HTTP/S / SOCKS proxy support to avoid invalid cookies

**Comment by ngosang** [comment]
This has to be fixed in FlareSolverr first => https://github.com/FlareSolverr/FlareSolverr/issues/26

**Comment by JReverse** [comment]
@ngosang Yes but there's also another alterative me installing squid on the docker machine that's why I asked you to add proxy support i think that'd work unsure tho lmk thats why i asked for you to add support in the lib

**Comment by mastercho** [comment]
As i see there is proxy support which is forket from CloudProxy project but not documented in both projects, if you add it to FlareSolverrSharp and document it in Solverr itself should be fine. I test and seems is fine on FlareSolverr

**Comment by ngosang** [comment]
Implemented in 2ee429b8ee5d2f16c8251a74e9985a106842d16f
Release 2.0.0 is out!
